pkg/config/initoptions_test.go

```diff
@@ -31,8 +81,15 @@ func TestComplete(t *testing.T) {
 			args:    []string{"foo"},
 			isError: true,
 		},
+		"More than one namespace should fail": {
+			args:    []string{d1},
+			isError: true,
+		},
+		//"No namespace set is fine": {
```

Comment:
Should this be commented out?

Reply:
removed since the test will stop and writing, the e2e test validated this case.